### DISCUSSION TITLE
Use rich instead of manual coloring with colorama

### DISCUSSION THREAD

**Discussion by sbordeyne** [ThioJoe/YT-Spammer-Purge#501] [Ideas]
`rich` is a python package which is able to produce rich text to the CLI. It natively supports the entire unicode character set, is able to print colors, bold, underline, strikethrough, animated text, progress bars, spinners, neat table or panels. It's a godsent in terms of formatting. It also includes a print function, which can supplant the standard print with additionnal features to color text, for instance : 

```py
from rich import print

print(f"      4. Scan a specific [magenta]community post[/] (Experimental)")
```

instead of the existing `print(f"      4. Scan a specific {F.LIGHTMAGENTA_EX}community post{S.R} (Experimental)")`

**Comment by ThioJoe** [comment]
The problem is the program is basically limited by what the default windows console can display
